src/Type/Php/JsonThrowOnErrorDynamicReturnTypeExtension.php

```diff
@@ -109,7 +115,7 @@ private function isForceArray(FuncCall $funcCall, Scope $scope): bool
 		}
 
 		$secondArgType = $scope->getType($args[1]->value);
-		$secondArgValue = $secondArgType instanceof ConstantScalarType ? $secondArgType->getValue() : null;
+		$secondArgValue = count($secondArgType->getConstantScalarValues()) === 1 ? $secondArgType->getConstantScalarValues()[0] : null;
```

Comment:
`$secondArgType->getConstantScalarValues()` with more than one value should also be properly handled here